dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-chunjun.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ref, onMounted, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import type { IJsonItem } from '../types'
+import { useChunjunDeployMode } from './'
+
+export function useChunjun(model: { [field: string]: any }): IJsonItem[] {
+  const { t } = useI18n()
+
+  const jsonEditorSpan = ref(0)
+  const customParameterSpan = ref(0)
+
+  const initConstants = () => {
+    if (model.customConfig) {
+      jsonEditorSpan.value = 24
+      customParameterSpan.value = 24
+    } else {
+      jsonEditorSpan.value = 0
+      customParameterSpan.value = 0
+    }
+  }
+
+  onMounted(() => {
+    initConstants()
+  })
+
+  watch(
+    () => model.customConfig,
+    () => {
+      initConstants()
+    }
+  )
+
+  return [
+    {
+      type: 'switch',
+      field: 'customConfig',
+      value: true,
+      name: t('project.node.chunjun_custom_template'),
+      props: {
+        disabled: true
```

Comment:
Why do you set disabled to true?

Reply:
Default support custom template

Reply:
Instead of setting the default value, it will cause the Switch to be disabled and cannot be switched. You should delete it. @tracehh

Reply:
OK, get it, thanks very much @devosend